internal/constants/defaults.go

```diff
@@ -9,4 +9,7 @@ const TerraformBinName = "terraform"
 const TerraformBinVersion = "0.12.18"
 const TerraformBinDownloadURLFormat = "https://releases.hashicorp.com/terraform/%s/terraform_%s_%s_%s.zip"
 
-const DCEBackendVersion = "0.23.0"
+// Default version of DCE to deploy, using `dce system deploy`
+const DefaultDCEVersion = "0.23.0"
```

Comment:
Note this PR does not yet include a DCE version upgrade.
I'll put that in a separate PR